layout/_third-party/analytics/index.swig

```diff
@@ -1,3 +1,4 @@
+{% include 'busuanzi-counter.swig' %}
 {% include 'facebook-sdk.swig' %}
 {% include 'vkontakte-api.swig' %}
 {% include 'google-analytics.swig' %}
```

Comment:
Also, if u move `tencent`, `cnzz` and other from index to widget, u must delete it from index, right?